config/rbac/dbaasprovider_viewer_role.yaml

```diff
@@ -0,0 +1,20 @@
+# permissions for end users to view dbaasproviders.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dbaasprovider-viewer-role
+rules:
+  - apiGroups:
+      - dbaas.redhat.com
+    resources:
+      - dbaasproviders
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - dbaas.redhat.com
+    resources:
+      - dbaasproviders/status
+    verbs:
+      - get
```

Comment:
Should we defining this as part of the partner operator or as part of DBaaS operator?

Reply:
removed, this  ClusterRole will be part of the DBaaS operator. I have tested and it works without this role.